tests/FSharp.Compiler.ComponentTests/xunit.runner.json

```diff
@@ -1,7 +1,6 @@
 ﻿{
-    "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
-    "appDomain": "ifAvailable",
-    "shadowCopy": false,
-    "parallelizeTestCollections": false,
-    "maxParallelThreads": 1
+  "$schema": "https://xunit.net/schema/current/xunit.runner.schema.json",
+  "appDomain": "denied",
```

Comment:
Switching off appdomain creation for net472 (this setting is not used for netcore) is needed to properly install the TestConsole as assembly fixture. Turns out none of our tests require it.